kasa/iot/iotbulb.py

```diff
@@ -204,6 +205,22 @@ def __init__(
         self.add_module("countdown", Countdown(self, "countdown"))
         self.add_module("cloud", Cloud(self, "smartlife.iot.common.cloud"))
 
+    async def _initialize_features(self):
+        await super()._initialize_features()
+
+        if bool(self.sys_info["is_dimmable"]):  # pragma: no branch
```

Comment:
The reason for keeping this on the iotbulb and iotdimmer is because `brightness` and `set_brightness` don't exist on the iotdevice so setting them as `attribute_getter` etc seemed flaky.  It also kept the logic of which thing to check contained in each class because the dimmer and bulb check for `is_dimmable` and `brightness` elsewhere.  What do you think?

Reply:
Fair enough :+1: I'm wondering if we should have a set of standard actions that we can "plug" in, with standardized getter/setter/icon/names to avoid repeating ourselves for cases like power/brightness/color_temp/color etc.?